packages/frontend/src/translations/en.global.json

```diff
@@ -195,9 +195,12 @@
         "allow": "Allow",
         "approve": "Approve",
         "authorizing": "Authorizing",
+        "backToSwap": "Back to Swap",
         "browseApps": "Browse Apps",
         "buy": "Buy",
         "cancel": "Cancel",
+        "backToSwap": "Back to Swap",
```

Comment:
Is the key duplicate with LINE 198? just noticed this since I want to help Ruby to add Chinese translation.